### PR TITLE
Use rules_pkg to make the skylib tarball for a release.

### PR DESCRIPTION
**Pull request by aiuto**
The addition of a rolloing changelog is debatable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+**New Features**
+
+-   common_settings.bzl: Standard data types for user defined build
+    configuration. Common scalar build settings for rules to use so they don't
+    recreate them locally. This fulfills part of the SBC design doc:
+    https://docs.google.com/document/d/1vc8v-kXjvgZOdQdnxPTaV0rrLxtP2XwnD2tAZlYJOqw/edit#bookmark=id.iiumwic0jphr
+-   selects.bzl: Add config_setting_group for config_setting AND/OR-chaining
+    Implements
+    https://github.com/bazelbuild/proposals/blob/master/designs/2018-11-09-config-setting-chaining.md.
+-   Make sets.bzl point to new_sets.bzl instead of old_sets.bzl new_sets.bzl and
```
Comment (fweikert) [inline CHANGELOG.md]: Remove "new_sets.bzl"?

Comment (fweikert) [inline CHANGELOG.md]: Missing dot

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    old_sets.bzl should be removed in the following skylib release.
+
+-   run_binary: runs an executable as an action
+
+    -   This rule is an alternative for genrule(): it canrun a binary with the
```
Comment (fweikert) [inline CHANGELOG.md]: missing space

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    On Linux/macOS/non-Windows, the test compares files using 'diff'.
+
+    On Windows, the test compares files using 'fc.exe'. This utility is
+    available on all Windows versions I tried (Windows 2008 Server, Windows 2016
+    Datacenter Core).
+
+    See https://github.com/bazelbuild/bazel/issues/5508,
+    https://github.com/bazelbuild/bazel/issues/4319
+
+-   maprule: move functionality to maprule_util.bzl maprule_util.bzl will
```
Comment (fweikert) [inline CHANGELOG.md]: Missing period

File: CHANGELOG.md (hunk 2 above)
Comment (aiuto) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (aiuto) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (aiuto) [reply CHANGELOG.md]: done

**Pull request merged**